### PR TITLE
Eclipse OpenJ9 needs to use the trademark symbol

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/986

Added trademark symbol to the first instance of "Eclipse OpenJ9" (2nd instance if the first instance was in a heading/title) in all topics.

Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the documents. Please check and confirm. Thanks!

File: docs/tool_jmap.md [hunk 1]
```diff
@@ -1,79 +1,77 @@
...
--->
-
-# Java memory map (`jmap`) tool
-
-Use the `jmap` tool to get memory information for a particular Java&trade; process, or list of processes. The tool shows statistics about classes on the heap, including the number of objects and their aggregate size. The command syntax is as follows:
-
-    jmap [<option>] [<vmid>]
-
-`<vmid>` is the Attach API virtual machine identifier for the Java process. This ID is typically the same as
-the operating system *process ID*, unless you specified the `-Dcom.ibm.tools.attach.id` system property when you started the process.
```
Comment (pshipton) [inline docs/tool_jmap.md:32]: How come this entire file is showing as a difference, and this point was removed?

Comment (pshipton) [reply docs/tool_jmap.md:32]: Nm, I see it wasn't removed, just re-formatted.

**Review by pshipton — APPROVED** [review]

**Pull request merged**